### DISCUSSION TITLE
Another notice - Fairly Important

### DISCUSSION THREAD

**Discussion by akabutnicer** [3kh0/ext-remover#1358] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
i shouldn't be saying this at 1 AM, but everybody listen up. when you post an exploit in these discussions, append `- webview` to the title if your exploit contains either one of these things:
chrome apps
kiosk apps
playstore app

if your exploit includes those but the term 'webview' is incorrectly applied, we will only then correct you and revoke said tag.

 people shouldn't have to correct you for your mistakes, so please follow these rules from now on.

**Comment by ghost** [comment]
thanks sir
